src/chrome/content/rules/General-Electric.xml

```diff
@@ -1,6 +1,6 @@
 <ruleset name="General Electric (partial)">
 
-	<target host="files.*.geblogs.com"/>
+	<target host="*.geblogs.com"/>
```

Comment:
Could you also add some (we require at least 3) test urls and a hint, that the blogs itself do not support HTTPS.

Reply:
Doesn't rule itself give such a hint? Its regexp clearly covers only files.*.geblogs.com.

Reply:
This is the last PR needed before we can forbid wildcards in the middle. @RReverser, are you interested in addressing @J0WI's comments (ie add `test url`s and a comment *why* other subdomains do not support HTTPS)?

If you don't have time / don't want to work on this, I can take over.